Add documentation to disable table of content filter (#261)
Yoast/developer: docs/customization/yoast-seo-premium/disable-table-of-content-block.md

---
id: disable-table-of-content-block

> title: "Yoast SEO Premium: Disable Table of Content Block"

Reviewer: ```suggestion
title: "Yoast SEO Premium: Disable the Table of Content Block"
```

> sidebar_label: Disable Table of Content Block

Reviewer: ```suggestion
sidebar_label: Disable the Table of Content Block
```

---

> We added a filter in Yoast SEO Premium version 21.5 to allow disabling the `Table of Content` block.
> 
> Used to prevent auto-generation of HTML anchors for headings when `Table of Content` block is registered.
> 
> Please add the following code to your theme's `functions.php` file.

Reviewer: ```suggestion
We added a filter in Yoast SEO Premium 21.5 to allow disabling the `Table of Content` block. This is useful to prevent the auto-generation of HTML anchors for headings when `Table of Content` block is registered, if you are not using the TOC block at all.

Add a snippet like this one in your theme's `functions.php` file:
```
